### PR TITLE
AC-560 - Fixed text overflowing in ListView items

### PR DESCRIPTION
**Pull request by sanjulamadurapperuma**
## Description of what I changed
Changed layout heights in fragment_patient_charts.xml and text size and weight sum in list_vital_group.xml in order to prevent text overlapping in ListView items.

## Issue I worked on
This is a mild cosmetic bug in the Android Application
JIRA Issue: https://issues.openmrs.org/browse/AC-560

## Checklist: I completed these to help reviewers :)
- [x] My pull request only contains **ONE single commit**

- [ ] I have **added tests** to cover my changes. (Only existing code was refactored)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: openmrs-client/src/main/res/layout/fragment_patient_charts.xml [hunk 1]
```diff
@@ -20,18 +20,18 @@
...
+    <!--android:divider="@null"-->
+    <!--android:childDivider="@drawable/exp_list_divider"/>-->
 
     <ListView android:id="@+id/vitalList"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:dividerHeight="1dp"
         android:divider="@null"
-       />
+        />
```
Comment (deepak140596) [inline openmrs-client/src/main/res/layout/fragment_patient_charts.xml]: remove this space

File: openmrs-client/src/main/res/layout/fragment_patient_charts.xml [hunk 2]
```diff
@@ -20,18 +20,18 @@
...
-        <!--android:layout_width="match_parent"-->
-        <!--android:layout_height="match_parent"-->
-        <!--android:dividerHeight="1dp"-->
-        <!--android:divider="@null"-->
-        <!--android:childDivider="@drawable/exp_list_divider"/>-->
+    <!--android:layout_width="match_parent"-->
+    <!--android:layout_height="match_parent"-->
+    <!--android:dividerHeight="1dp"-->
+    <!--android:divider="@null"-->
+    <!--android:childDivider="@drawable/exp_list_divider"/>-->
```
Comment (deepak140596) [inline openmrs-client/src/main/res/layout/fragment_patient_charts.xml]: if these lines are not being used, remove these lines.

File: openmrs-client/src/main/res/layout/fragment_patient_charts.xml (hunk 1 above)
Comment (sanjulamadurapperuma) [reply openmrs-client/src/main/res/layout/fragment_patient_charts.xml]: Done. Thanks for the advice

File: openmrs-client/src/main/res/layout/fragment_patient_charts.xml (hunk 2 above)
Comment (sanjulamadurapperuma) [reply openmrs-client/src/main/res/layout/fragment_patient_charts.xml]: Done

**Comment by f4ww4z** [comment]
@sanjulamadurapperuma Did you see the [material layout guidelines](https://material.io/design/layout/understanding-layout.html#usage)? Also always stick to our PR guidelines: https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by sanjulamadurapperuma** [comment]
> 
> 
> @sanjulamadurapperuma Did you see the [material layout guidelines](https://material.io/design/layout/understanding-layout.html#usage)? Also always stick to our PR guidelines: https://wiki.openmrs.org/display/docs/Pull+Request+Tips

Sure, I'll take a look at that. And yes I will follow the PR guidelines in the future. Thanks a lot for the advice!

**Comment by f4ww4z** [comment]
@sanjulamadurapperuma Closing this PR to reflect the [issue's change of focus](https://issues.openmrs.org/browse/AC-560).

**Pull request closed without merging**